### ISSUE TITLE
[BUG] AMBER Process getPressure would not recognise 

### ISSUE THREAD

**Issue by xiki-tempula** [OpenBioSim/biosimspace#84]
**Describe the bug**
The first frame of my amber output says `PRESS =********`, which is ignored in Process.getPressure, this would mean that getPressure would return a different number of elements compared with other metrics.

**To Reproduce**

```
import BioSimSpace.Sandpit.Exscientia as BSS
mol = BSS.Parameters.openff_unconstrained_2_0_0('C').getMolecule()
process = BSS.Process.Amber(mol.toSystem(), BSS.Protocol.FreeEnergyEquilibration(), work_dir='.')
print(len(self.getPressure(True)))
print(len(self.self.getVolume(True)))
```

There is 26 numbers in Volume but only 25 in pressure.

**Expected behavior**
Both of them give 26 numbers

**Input files**
[amber.out.zip](https://github.com/OpenBioSim/biosimspace/files/11567204/amber.out.zip)

**Comment by lohedges** [comment]
Just to say that I've not forgotten this and hope to add a fix in time for the 2023.3.0 release.

**Comment by xiki-tempula** [comment]
great thanks
